ui/apps/everest/.e2e/release/init-deploy.e2e.ts

```diff
@@ -0,0 +1,180 @@
+import { getClusterDetailedInfo } from '../utils/storage-class';
+import { resourcesStepCheck } from '../db-cluster/db-wizard/create-db-cluster/steps/resources-step';
+import {
+  moveForward,
+  submitWizard,
+  populateBasicInformation,
+} from '../utils/db-wizard';
+import { EVEREST_CI_NAMESPACES } from '../constants';
+import { waitForStatus, waitForDelete } from '../utils/table';
+
+test.describe.configure({ retries: 0 });
+
+[
+  { db: 'psmdb', size: 1 },
+  { db: 'psmdb', size: 3 },
+  { db: 'pxc', size: 1 },
+  { db: 'pxc', size: 3 },
+  { db: 'postgresql', size: 1 },
+  { db: 'postgresql', size: 3 },
+].forEach(({ db, size }) => {
+  test.describe(
+    'Init deploy workflow',
+    {
+      // Create/stop/resume/delete cluster in different configurations
+      tag: '@release',
+    },
+    () => {
+      const clusterName = `${db}-${size}-deploy`;
+
+      let storageClasses = [];
+      let namespace = EVEREST_CI_NAMESPACES.EVEREST_UI;
+
+      test.beforeAll(async ({ request }) => {
+        const token = await getTokenFromLocalStorage();
+
+        const { storageClassNames = [] } = await getClusterDetailedInfo(
+          token,
+          request
+        );
+        storageClasses = storageClassNames;
+      });
+
+      test(`Cluster creation with ${db} and size ${size}`, async ({
+        page,
+        request,
+      }) => {
+        test.setTimeout(size * 120000);
+
+        expect(storageClasses.length).toBeGreaterThan(0);
+
+        await page.goto('/databases/new');
+        await page.getByTestId('toggle-button-group-input-db-type').waitFor();
+        await page.getByTestId('select-input-db-version').waitFor();
+
+        // basic info
+        await populateBasicInformation(
+          page,
+          db,
+          storageClasses[0],
+          clusterName
+        );
+        await moveForward(page);
+
+        // resources
+        await page
+          .getByRole('button')
+          .getByText(size + ' node')
+          .click();
+        await expect(page.getByText('Nº nodes: ' + size)).toBeVisible();
+        await resourcesStepCheck(page, 0.6, 1, 1, size);
+        await moveForward(page);
+
+        // backups
+        await moveForward(page);
+
+        // advanced
+        await moveForward(page);
+
+        // monitoring
+        await submitWizard(page);
+        await expect(
+          page.getByText('Awesome! Your database is being created!')
+        ).toBeVisible();
+
+        // go to db list and check status
+        await page.goto('/databases');
+        await waitForStatus(page, clusterName, 'Initializing', 15000);
+        await waitForStatus(page, clusterName, 'Up', 240000);
+
+        const response = await request.get(
+          `/v1/namespaces/${namespace}/database-clusters`,
+          {
+            headers: {
+              Authorization: `Bearer ${await getTokenFromLocalStorage()}`,
+            },
+          }
+        );
+
+        expect(response.ok()).toBeTruthy();
```

Comment:
the fact that the response is not ok is not really informative and we would need to debug the test to understand what is the error. For the response checking consider to use something like this
https://github.com/percona/everest/blob/3f2baae7a2432c4b957210365690a22371bd357b/api-tests/tests/helpers.ts#L11-L16 
so if the response is not ok we would see the error message right away

Reply:
Thank you! I have applied this.